qemu/tests/cfg/nvdimm.cfg

```diff
@@ -92,3 +93,13 @@
         - nvdimm_hotplug:
             del mem_devs
             target_mems = mem1
+        - nvdimm_nvml:
+            depends_pkgs = ndctl-devel daxctl-devel pandoc
+            nvml_test = yes
+            nvml_dir = /tmp/nvml
+            get_nvml = "git clone --depth=1 https://github.com/pmem/nvml.git ${nvml_dir}"
+            config_nvml = "cd ${nvml_dir}; make -j4"
+            config_nvml += ";cp src/test/testconfig.sh.example src/test/testconfig.sh"
+            config_nvml += ";echo 'PMEM_FS_DIR_FORCE_PMEM=1' >> src/test/testconfig.sh"
+            config_nvml += ";echo 'PMEM_FS_DIR=${mount_dir}' >> src/test/testconfig.sh"
```

Comment:
Please help to add "make EXTRA_CFLAGS="-USDS_ENABLED" before "make check" on ppc, test case was refined already.